File: android/src/main/java/expo/modules/alipay/ExpoAlipayModuleTypes.kt
package expo.modules.alipay

import expo.modules.kotlin.records.Field
import expo.modules.kotlin.records.Record

class PayOptions: Record {
    @Field
    var orderInfo: String = ""
    @Field
    var scheme: String? = null
    @Field
    var universalLink: String? = null
}

class AuthOptions: Record {
    @Field
    var authInfo: String = ""
    @Field
    var scheme: String? = null
    @Field
    var universalLink: String? = null
}

File: ios/ExpoAlipayModuleTypes.swift
//
//  ExpoAlipayModuleTypes.swift
//  ExpoAlipay
//
//  Created by Aron on 2025/7/26.
//

import Foundation
import ExpoModulesCore


struct PayOptions: Record {
    @Field
    var orderInfo: String = ""
    @Field
    var scheme: String?
    @Field
    var universalLink: String?
}

struct AuthOptions: Record {
    @Field
    var authInfo: String = ""
    @Field
    var scheme: String?
    @Field
    var universalLink: String?
}